src/meta/src/barrier/command.rs

```diff
@@ -71,6 +72,15 @@ pub struct Reschedule {
     pub actor_splits: HashMap<ActorId, Vec<SplitImpl>>,
 }
 
+#[derive(Debug, Clone)]
+pub struct ReplaceTableCommand {
```

Comment:
```suggestion
pub struct ReplaceTablePlan {
```
Unify it with `Command::ReplaceTable`, and better not name it as command, but I don't have any better naming. 🥵